datahub-web-react/src/app/ingest/source/executions/ExecutionRequestDetailsModal.tsx

```diff
@@ -132,7 +132,12 @@ export const ExecutionDetailsModal = ({ urn, visible, onClose }: Props) => {
         undefined;
 
     const recipeJson = data?.executionRequest?.input.arguments?.find((arg) => arg.key === 'recipe')?.value;
-    const recipeYaml = recipeJson && YAML.stringify(JSON.parse(recipeJson), 8, 2);
+    let recipeYaml: string;
+    try {
+        recipeYaml = recipeJson && YAML.stringify(JSON.parse(recipeJson), 8, 2);
+    } catch (e) {
+        recipeYaml = '';
```

Comment:
we can also just fall back to 
```suggestion
        recipeYaml = recipeJson;
```

Reply:
I'd probably want to alter the expanding logic if I supported this, so going to hold off on making this change for simplicity. If this is a common occurrence we can revisit